src/libraries/Common/src/System/Security/Cryptography/RSAAndroid.cs

```diff
@@ -306,7 +306,7 @@ public override void ImportParameters(RSAParameters parameters)
                 ValidateParameters(ref parameters);
                 ThrowIfDisposed();
 
-                if (parameters.Exponent == null || parameters.Modulus == null)
+                if (parameters.Exponent == null || parameters.Modulus.AsSpan().IndexOfAnyExcept((byte)0) < 0)
```

Comment:
The implicit `null` to an empty span works here, so the null check would be redundant with this check. Really what we are looking for is "Does modulus contain anything that is not zero. If not, the parameters are invalid."